twostep.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Plugin Name: VIP Force Two Step
```

Comment:
Naming consistency: we use the terms `two step`, 2fa`, and `two factor` all interchangibly. We should probably pick one and stick to it.